Add 05 to the student list

```diff
@@ -8,3 +8,4 @@
 | № | Имя Фамилия     | Любимый фильм                          |
 | - |:---------------:| --------------------------------------:|
 | 0 | Ава Катушка     | Три идиота                             |
+| 5 | Василина Жонкина     | Автостопом по галактике                             |
```

Comment:
Please fix the layout of the table. All "|" should be in the same line (you must move it in all the table). 
And also incorporate commits from other people into your submission (you are not the number 5 in this table).